src/routes/safe/store/actions/transactions/fetchTransactions/index.ts

```diff
@@ -7,10 +7,13 @@ import { loadOutgoingTransactions } from './loadOutgoingTransactions'
 
 import { addOrUpdateCancellationTransactions } from 'src/routes/safe/store/actions/transactions/addOrUpdateCancellationTransactions'
 import { addOrUpdateTransactions } from 'src/routes/safe/store/actions/transactions/addOrUpdateTransactions'
+import { Dispatch } from 'redux'
+import { loadAllTransactions } from './loadAllTransactions'
+import { addNewTransactions } from '../../addNewTransactions'
 
 const noFunc = () => {}
 
-export default (safeAddress: string) => async (dispatch) => {
+export default (safeAddress: string) => async (dispatch: Dispatch): Promise<void> => {
```

Comment:
should we also add parameters for pagination?

Reply:
I just remove the usage of `loadAllTransactions` here